queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name from animals WHERE neutered = true AND escape_attempts < 3;
```

Comment:
- [ ] Dear @Tamunokuro, here we are asked to list the name of all animals born between 2016 and 2019. which means we need to find `date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12'` inclusive. Please update your query accordingly.

Reply:
well resolved! ✔️